types/lockfile-info/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for lockfile-info 1.0
+// Project: https://github.com/ljharb/lockfile-info
+// Definitions by: Jordan Harband <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/lockfile-info/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me